bom/pom.xml

```diff
@@ -23,4 +23,62 @@
     <module>openhab-addons</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-antrun-plugin</artifactId>
+          <version>1.8</version>
+          <inherited>false</inherited>
+          <executions>
+            <execution>
+              <id>create-bom</id>
+              <goals>
+                <goal>run</goal>
+              </goals>
+              <configuration>
+                <target>
+                  <copy file="${basedirRoot}/../../bundles/pom.xml" tofile="${basedirRoot}/../../bom/openhab-addons/pom.xml" overwrite="true"/>
+                  <!-- rewrite footer -->
+                  <replaceregexp file="${basedirRoot}/../../bom/openhab-addons/pom.xml" match="/modules[\s\S]*dependencies>" replace="/dependencies>" />
+                  <!-- reqrite header header -->
+                  <replaceregexp file="${basedirRoot}/../../bom/openhab-addons/pom.xml" match="\S*parent[\s\S]*modules>\S*" replace="header" />
+                  <replace file="{basedirRoot}/../../bom/openhab-addons/pom.xml">
+                    <replacetoken>header</replacetoken>
+                    <replacevalue><![CDATA[<parent>
+    <groupId>org.openhab.addons.bom</groupId>
+    <artifactId>org.openhab.addons.reactor.bom</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
```

Comment:
Can we make this dynamic this will make releasing though

Reply:
This can be replaced by the project.version, true.

Reply:
It's the parent reference so not trivial though

Reply:
since maven replaces it with the value that is valid during the run, it's easy.